kubernetes/windows/main.ps1

```diff
@@ -263,6 +263,27 @@ function Generate-Certificates {
     C:\\opt\\omsagentwindows\\certgenerator\\certificategenerator.exe
 }
 
+function Bootstrap-CACertificates {
+    try {
+        # This is required when the root CA certs are different for some clouds.
+        $caCerts=Invoke-WebRequest 'http://168.63.129.16/machine?comp=acmspackage&type=cacertificates&ext=json' -UseBasicParsing | ConvertFrom-Json
```

Comment:
since us{nat;sec} clouds are stringent on security, wondering why http instead of https here?

Reply:
here too..

Reply:
To answer you qs - this was the url provided by the compute team. So i didnt want to make any changes to that.

Reply:
> since new clouds are stringent on security, wondering why http instead of https here?



> us{nat;sec}

Reply:
I think, we need to understand why they are not supporting https. I am pretty sure they might have support https or plan to enable this when we raise this question.

Reply:
@ganga1980 - You can find more information here - https://docs.microsoft.com/en-us/azure/virtual-network/what-is-ip-address-168-63-129-16#scope-of-ip-address-1686312916

This is an IP address owned by Microsoft and this is an excerpt from the doc - " The communication between this special IP address and the resources is safe because only the internal Azure platform can source a message from this IP address"